taskcluster/script.py

```diff
@@ -201,10 +149,13 @@ def main():
     except subprocess.CalledProcessError as e:
         print('{} attempting netstat'.format(e))
 
-    print('payload.py exitcode {}'.format(rc))
+    print('script.py exitcode {}'.format(rc))
     if rc == 0:
         return 0
     return 1
 
 if __name__ == "__main__":
-    sys.exit(main())
+    parser = argparse.ArgumentParser()
```

Comment:
I'd rather not have these in global scope. Let's move the parser stuff into main and just do it there. Everything else looks great.

Reply:
Changed.